ludwig/experiment.py

```diff
@@ -287,34 +287,34 @@ def experiment(
 
 
 def kfold_cross_validate(
-        k_fold,
-        model_definition=None,
-        model_definition_file=None,
-        data_csv=None,
-        output_directory='results',
-        random_seed=default_random_seed,
-        skip_save_k_fold_split_indices=False,
-        **kwargs
+    k_fold,
+    model_definition=None,
+    model_definition_file=None,
+    data_csv=None,
```

Comment:
this should work also with dataframes

Reply:
Added support for pandas dataframes.  updated function signature
```
def experiment(
        model_definition,
        model_definition_file=None,
        data_df=None,
        data_train_df=None,
        data_validation_df=None,
        data_test_df=None,
        data_csv=None,
        data_train_csv=None,
        data_validation_csv=None,
        data_test_csv=None,
        data_hdf5=None,
        data_train_hdf5=None,
        data_validation_hdf5=None,
        data_test_hdf5=None,
```
and
```
def full_experiment(
        model_definition,
        model_definition_file=None,
        data_df=None,
        data_train_df=None,
        data_validation_df=None,
        data_test_df=None,
        data_csv=None,
        data_train_csv=None,
        data_validation_csv=None,
        data_test_csv=None,
        data_hdf5=None,
        data_train_hdf5=None,
        data_validation_hdf5=None,
        data_test_hdf5=None,
```